pagopa/pn-frontend
# Migrated to pg api for tos retrieve and acceptance - pn-12392

## Short description
When a PG enables the SERCQ SEND, it has to accept the linked tos and privacy. For the PG there are specific api to call

## List of changes proposed in this pull request
- Updated bff dependencies
- Updated api called during retrieve and acceptance of tos and dataprivacy

## How to test
Login with PG (no DivinaCommedia) -> enable SERCQ SEND -> check everything is ok
